core-bundle/src/Exception/ServiceUnavailableException.php

```diff
@@ -12,12 +12,8 @@
 
 namespace Contao\CoreBundle\Exception;
 
-trigger_deprecation('contao/core-bundle', '4.1', 'Using the "Contao\CoreBundle\Exception\ServiceUnavailableException" class has been deprecated and will no longer work in Contao 5.0. Use the "Lexik\Bundle\MaintenanceBundle\Exception\ServiceUnavailableException" class instead.');
+use Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException;
```

Comment:
IMHO we should keep the deprecation and update it to say `use the Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException class instead.`